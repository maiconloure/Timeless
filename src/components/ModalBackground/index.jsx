import React from 'react';
import { Background } from '../EditModalStyles';

const ModalBackground = ({ show }) => {
  return <>{show && <Background />}</>;
};

export default ModalBackground;
