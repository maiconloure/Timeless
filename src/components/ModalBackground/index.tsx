import React from 'react';
import { Background } from '../EditModalStyles';

interface Props {
  show: boolean;
}

const ModalBackground = ({ show }: Props) => {
  return <>{show && <Background />}</>;
};

export default ModalBackground;
