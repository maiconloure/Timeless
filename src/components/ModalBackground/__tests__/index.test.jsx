import React from 'react';
import ModalBackground from '../index';
import renderer from 'react-test-renderer';

describe('Snapshot', () => {
  it('renders', () => {
    const tree = renderer.create(<ModalBackground show={true} />).toJSON();

    expect(tree).toMatchSnapshot();
  });

  it('not renders', () => {
    const tree = renderer.create(<ModalBackground show={false} />).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
