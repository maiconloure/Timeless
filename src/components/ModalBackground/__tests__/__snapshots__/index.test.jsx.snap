// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot not renders 1`] = `null`;

exports[`Snapshot renders 1`] = `
<div
  className="sc-efHXLn dAUWVC"
/>
`;
